src/meta-srv/src/error.rs

```diff
@@ -840,6 +848,13 @@ pub enum Error {
         #[snafu(source(from(common_config::error::Error, Box::new)))]
         source: Box<common_config::error::Error>,
     },
+
+    #[snafu(display("Region: {} leader peer is not found", region_id))]
+    RegionLeaderNotFound {
```

Comment:
Looks like this variant is not used anywhere?

Reply:
Removed